shared-bindings/audiomixer/MixerVoice.c

```diff
@@ -107,6 +107,34 @@ MP_PROPERTY_GETSET(audiomixer_mixervoice_level_obj,
     (mp_obj_t)&audiomixer_mixervoice_get_level_obj,
     (mp_obj_t)&audiomixer_mixervoice_set_level_obj);
 
+//|     loop: bool
+//|     """Get or set the loop status of the currently playing sample."""
+static mp_obj_t audiomixer_mixervoice_obj_get_loop(mp_obj_t self_in) {
+    return mp_obj_new_bool(common_hal_audiomixer_mixervoice_get_loop(self_in));
+}
+MP_DEFINE_CONST_FUN_OBJ_1(audiomixer_mixervoice_get_loop_obj, audiomixer_mixervoice_obj_get_loop);
+
+static mp_obj_t audiomixer_mixervoice_obj_set_loop(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
+    enum { ARG_loop };
+    static const mp_arg_t allowed_args[] = {
+        { MP_QSTR_loop,     MP_ARG_BOOL | MP_ARG_REQUIRED, {} },
+    };
```

Comment:
for a property setter, there's no need to support kwargs. I notice that you were probably just copying the style of `audiomixer_mixervoice_obj_set_level` within this same file so what you wrote is understandable, but would be better to write it similar to this more usual setter:
```c
static mp_obj_t synthio_synthesizer_obj_set_envelope(mp_obj_t self_in, mp_obj_t envelope) {
    synthio_synthesizer_obj_t *self = MP_OBJ_TO_PTR(self_in);
    check_for_deinit(self);
    synthio_synth_envelope_set(&self->synth, envelope);
    return mp_const_none;
}
MP_DEFINE_CONST_FUN_OBJ_2(synthio_synthesizer_set_envelope_obj, synthio_synthesizer_obj_set_envelope);
```

Reply:
Ah, I did wonder about that, and yes, very much a copy-and-paste job.  Fixed in 372759f728.